pdr_backend/lake/etl.py

```diff
@@ -164,38 +164,54 @@ def _get_max_timestamp_values_from(
             "SELECT '{}' as table_name, MAX(timestamp) as max_timestamp FROM {}"
         )
 
-        queries = [
-            max_timestamp_query.format(
-                get_table_name(table_name, table_type),
-                get_table_name(table_name, table_type),
+        all_db_tables = PersistentDataStore(
```

Comment:
please review my PR before working through this
i believe it addresses a lot of this stuff

Reply:
As I see the `_get_max_timestamp_values_from` method does not have any changes on your PR